README.md

```diff
@@ -145,21 +145,12 @@ second argument of `new`. Terrapin assumes that you will not be manually
 passing user-generated data to that argument and will be using it as a template
 for your command line's structure.
 
-## POSIX Spawn
-
-You can potentially increase performance by installing [the posix-spawn
-gem](https://rubygems.org/gems/posix-spawn). This gem can keep your
-application's heap from being copied when forking command line
-processes. For applications with large heaps the gain can be
-significant. To include `posix-spawn`, simply add it to your `Gemfile` or,
-if you don't use bundler, install the gem.
-
 ## Runners
 
 Terrapin will attempt to choose from among 3 different ways of running commands.
```

Comment:
Since we just have two runners now this section doesn't seem to provide as much value as it did before. It also references ruby 1.8 and 1.9, which are pretty ancient history at this point. I took a stab at rewriting this section to be a little more current. How would you feel about replacing this section with something along these lines? 

---

## Runners

Terrapin will choose from among a couple different ways of running commands.
The simplest is `Process.spawn`, which is also the default. Terrapin can also
just use [backticks], so if for some reason you'd prefer that, you can
ask Terrapin to use that:

```ruby
Terrapin::CommandLine.runner = Terrapin::CommandLine::BackticksRunner.new
```

And if you really want to, you can define your own Runner, though I can't
imagine why you would.

[backticks]: https://ruby-doc.org/3.2.1/Kernel.html#method-i-60